src/responses/yuzu.json

```
@@ -13,6 +13,7 @@
       "legal": { "reply": "*yuzu* is legal, we don't support illegal activities. Dumping your purchased games and system files from your Switch is legal. Downloading them is not."},
       "building": { "reply": "Please refer to our building guides.\nWindows: <https://yuzu-emu.org/wiki/building-for-windows> \nLinux: <https://yuzu-emu.org/wiki/building-for-linux>"},
       "contributing": { "reply": "Contributing to the project is the best way to help the project move forward. If you are a developer, please refer to: \nSwitch reference guides: <https://github.com/yuzu-emu/yuzu/wiki/Switch-Hardware-and-Software> \nSwitch homebrew applications for testing: <https://github.com/yuzu-emu/yuzu/wiki/Switch-Homebrew> \nWiki, dedicated to Switch research: <http://switchbrew.org> \nyuzu contributing tips: <https://github.com/yuzu-emu/yuzu/blob/master/CONTRIBUTING.md>"},
+      "multiplayer": { "reply": "New to yuzu multiplayer? Check out our multiplayer guide here: <https://yuzu-emu.org/help/feature/multiplayer/>"},
```

```suggestion
      "multiplayer": { "reply": "Please refer to our __multiplayer setup guide__: <https://yuzu-emu.org/help/feature/multiplayer/>"},
```

Probably best to have it be the same as the Citra command, as we try to keep them equal.